pprint/test/src/test/pprint/AdvancedTests.scala

```diff
@@ -4,188 +4,188 @@ import utest._
 
 import scala.collection.SortedMap
 
-object AdvancedTests extends TestSuite{
```

Comment:
These tests should be preserved, e.g. under `src-0/test/pprint/AdvancedTests.scala` – see upickle for example: https://github.com/dotty-staging/upickle/tree/dotty-community-build/upickle/test